x-pack/plugins/event_log/server/es/documents.ts

```diff
@@ -10,10 +10,7 @@ import mappings from '../../generated/mappings.json';
 // returns the body of an index template used in an ES indices.putTemplate call
 export function getIndexTemplate(esNames: EsNames) {
   const indexTemplateBody: any = {
-    index_patterns: [esNames.indexPattern],
-    aliases: {
```

Comment:
I guess the aliases aren't really needed here, since it's already set up in the `index.lifecycle.rollover_alias` property below this?

I guess my only worry is that if it rolls over for some other reason, that didn't take the that ilm property into account, the aliases could get into some funky state.  Not sure if it even CAN roll over for some other reason.  :-).

One of the nice things about this version, is that if a rando user creates an index matching our template, it WON'T get aliased in for free, which is likely what we want to happen.

Reply:
From what I recall, with the alias definition here, there would be ambiguity finding the write index after a second one is created. After doing some research, it was recommended to only setup `is_write_index` on the initial index and ILM would then handle the changeover automatically on rollover.

Reference: https://discuss.elastic.co/t/index-lifecycle-management-does-not-point-to-index-error/211513/2